pvlib/test/test_pvsystem.py

```diff
@@ -770,17 +752,13 @@ def test_scale_voltage_current_power(sam_data):
     out = pvsystem.scale_voltage_current_power(data, voltage=2, current=3)
 
 
-def test_PVSystem_scale_voltage_current_power():
-    data = pd.DataFrame(
-        np.array([[2, 1.5, 10, 8, 12, 0.5, 1.5]]),
-        columns=['i_sc', 'i_mp', 'v_oc', 'v_mp', 'p_mp', 'i_x', 'i_xx'],
-        index=[0])
-    expected = pd.DataFrame(
-        np.array([[6, 4.5, 20, 16, 72, 1.5, 4.5]]),
-        columns=['i_sc', 'i_mp', 'v_oc', 'v_mp', 'p_mp', 'i_x', 'i_xx'],
-        index=[0])
+def test_PVSystem_scale_voltage_current_power(mocker):
+    data = None
     system = pvsystem.PVSystem(modules_per_string=2, strings_per_inverter=3)
+    m = mocker.patch(
+        'pvlib.pvsystem.scale_voltage_current_power', autospec=True)
```

Comment:
This test function uses a different pattern than the others. Unlike ``mocker.spy``, ``mocker.patch`` can only be used to test that something is called because it completely replaces the function/method under consideration. I chose this pattern here because I think the input data is a little more complicated than is worth maintaining in the method test. Could be changed, though.